### ISSUE TITLE
[BUG][okhttp-gson] can not generate javadoc

### ISSUE THREAD

**Issue by jmini** [OpenAPITools/openapi-generator#3299]
##### Description

Introduced with issue #2994 / PR #2995 the generated javadoc now contains `@http.response.details`. Example from our the samples:

https://github.com/OpenAPITools/openapi-generator/blob/51e7005373fdb55daa1c253f36fb6267a64a7bf7/samples/client/petstore/java/okhttp-gson/src/main/java/org/openapitools/client/api/StoreApi.java#L63-L68

When trying to generate the javadoc artifact for the project, you get tons of errors:

```
cd samples/client/petstore/java/okhttp-gson
mvn javadoc:javadoc
```

```
INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  4.419 s
[INFO] Finished at: 2019-07-08T10:44:48+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-javadoc-plugin:2.10.4:javadoc (default-cli) on project petstore-okhttp-gson: An error has occurred in JavaDocs report generation: 
[ERROR] Exit code: 1 - /____/openapi-generator/samples/client/petstore/java/okhttp-gson/src/main/java/org/openapitools/client/api/AnotherFakeApi.java:63: error: unknown tag: http.response.details
[ERROR]      * @http.response.details
[ERROR]        ^
[ERROR] /____/openapi-generator/samples/client/petstore/java/okhttp-gson/src/main/java/org/openapitools/client/api/AnotherFakeApi.java:117: error: unknown tag: http.response.details
[ERROR]      * @http.response.details
[ERROR]        ^
[ERROR] /____/openapi-generator/samples/client/petstore/java/okhttp-gson/src/main/java/org/openapitools/client/api/AnotherFakeApi.java:134: error: unknown tag: http.response.details
[ERROR]      * @http.response.details
[ERROR]        ^
```

##### openapi-generator version

4.0.2

##### Suggest a fix

We can keep the table, but remove the `@http.response.details` and just put "Response details:"

@saigiridhar21, @wing328 what do you think?

**Comment by wing328** [comment]
Yes please go ahead with the fix.

**Comment by jmini** [comment]
I would put the html table before the parameters section.

```java
    /**
     * Send-Message
     * <p>Http response details:</p>
     <table summary="Response Details" border="1">
        <tr><td> Status Code </td><td> Description </td><td> Response Headers </td></tr>
        <tr><td> 200 </td><td>  </td><td>  -  </td></tr>
     </table>
     * @param sendMessageRequestBody  (required)
     * @throws ApiException If fail to call the API, e.g. server error or cannot deserialize the response body
     */
    public void sendMessage(SendMessageRequestBody sendMessageRequestBody) throws ApiException
```

IMO it looks better (preview of the javadoc in Eclipse IDE):

<img width="660" alt="Screenshot 2019-07-08 at 11 07 51" src="https://user-images.githubusercontent.com/1222165/60798250-1360d400-a171-11e9-9eda-c74110333282.png">

**Comment by jmini** [comment]
Wait, never-mind the issue is how the maven build is configured.

When the `configuration` bloc is correctly defined, you get decent result:
<img width="903" alt="Screenshot 2019-07-08 at 11 19 24" src="https://user-images.githubusercontent.com/1222165/60798899-5c655800-a172-11e9-94c1-e9a7592a5d41.png">

**Comment by saigiridhar21** [comment]
@jmini Do you mean that the configuration block is incorrectly configured ?

**Comment by jmini** [comment]
Thank you for the quick feedback. I have proposed a fix in #3302. Feel free to review.
